### DISCUSSION TITLE
Copy StreamField content between pages

### DISCUSSION THREAD

**Discussion by hpoul** [wagtail/wagtail#8053] [Ideas]
Hello,
I was wondering whether there was some easy way to copy StreamField content between pages. Or to be more precise from a Page into a Snippet with the same StreamField type. The only way I have seen yet is to either `Duplicate` a single block to have it copy/paste into the same StreamField, or copy the whole Page. But It would be nice to be able to copy content into a different page type or a snippet.

I guess internally it's just JSON so I could probably do it through the database, but is there another way, preferably usable by editors. I have also not found any related Issue, did I miss anything?

thanks,
  Herbert

**Comment by DEWaller** [comment]
It's a feature request out customers commonly request especially those who have had their site migrated from another CMS where this feature was available.

**Comment by SebCorbin** [comment]
See
- https://github.com/wagtail/wagtail/issues/8339
